wagtail/admin/templates/wagtailadmin/shared/page_status_tag.html

```diff
@@ -1,14 +1,9 @@
 {% load i18n wagtailadmin_tags %}
 {% if page.live %}
     {% with page_url=page.url %}
-        {% trans "Current page status:" as status_hidden_label %}
```

Comment:
This file will be updated in #10157 so please maybe find another one or two files to update with this convention.